ev3dev/core.py

```diff
@@ -137,6 +138,22 @@ def matches(attribute, pattern):
             if all([matches(path + '/' + k, kwargs[k]) for k in kwargs]):
                 yield f
 
+
+def file_open(path):
```

Comment:
Private?

Reply:
Actually, I can undo this change, it is no longer needed.